test/validationRules/required-fields.js

```diff
@@ -11,6 +11,7 @@ const requiredFieldsTestCases = {
     "const x = gql`query { greetings { id, hello, foo } }`",
     "const x = gql`query { greetings { id ... on Greetings { hello } } }`",
     "const x = gql`fragment Name on Greetings { id hello }`",
+    "const x = gql`fragment Foo on FooBar { id, hello, foo }`",
```

Comment:
Added this test case to prevent a regression – it would throw an error if not for [the fix in `src/customGraphQLValidationRules.js` introduced by this PR](https://github.com/apollographql/eslint-plugin-graphql/pull/231/files#diff-95dbafa783634330b4c1ec80e94c9964R22).